scripts/updateContributors.mjs

```diff
@@ -0,0 +1,79 @@
+import simpleGit from 'simple-git';
+import { Octokit } from '@octokit/rest';
+import fs from 'node:fs';
+import path from 'path';
+import pMemoize from 'p-memoize';
+import LOST_CONTRIBUTORS from './lost-contributors.json' assert { type: 'json' };
+
+const IGNORED_COMMITS = ['433bbae4f1d4abb50a26306d6679a38ace5c8b78'];
+
+const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+
+const cache = new Map();
+
+const getUserName = pMemoize(
+  async (commit) => {
+    const { data: fetchedCommit } = await octokit.repos.getCommit({
+      owner: 'lucide-icons',
+      repo: 'lucide',
+      ref: commit.hash,
+    });
+    return fetchedCommit?.author?.login;
+  },
+  { cacheKey: ([commit]) => commit.author_email, cache }
+);
+
+const getContributors = async (file, includeCoAuthors) => {
+  const { all: commits } = await simpleGit().log(['--reverse', file]);
```

Comment:
Failsafe to make sure that the file path is interpreted as a file and not a branch or tag:
```suggestion
  const { all: commits } = await simpleGit().log(['--reverse', '--', file]);
```